scipy/stats/_boost/meson.build

```diff
@@ -6,7 +6,7 @@ if meson.get_compiler('cpp', native: true).sizeof('void*') > 4
 endif
 
 include = include_directories('include')
-inc_boost = include_directories('..' / '..' / '_lib' / 'boost')
+boost_dep = dependency('boost')
```

Comment:
Note that https://github.com/scipy/boost-headers-only is at Boost 1.75. Boost 1.76 was broken in a way that was not easy to figure out. If you want to do this unbundling (also in nixpkgs) then please use `1.75.0`.

Reply:
Right, the intention is indeed to unbundle. I think the patch can be mainlined if a fallback is specified, but I have not tried that yet.